javascript/react/src/verbs/Merge.base.tsx

```diff
@@ -35,6 +35,25 @@ export const MergeBase: React.FC<
 				current: step.args.delimiter,
 				onChange: (s, val) => (s.args.delimiter = val as string),
 			},
+			{
```

Comment:
On main I added the ability to collapse extra params into an "advanced" expand/collapse section. I think a bunch of these could go in there to reduce clutter on the ux, particularly related to unhot stuff that will be less commonly used. You can pass `advanced: true` to the control generator functions (check out VerbForm). For this verb I would suggest the following get pushed into advanced: unhot, keepOriginalColumns, prefix.